syntax = "proto3";

import "trade/vo/futures/FuturesOrderVo.proto";
option
java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response.futures";
option java_outer_classname = "FuturesQueryRealEntrustListResponseProto";
message FuturesQueryRealEntrustListResponse {
	repeated FuturesOrderVo data = 1;
}
